index.bs

```diff
@@ -815,6 +815,12 @@ An attribution source is a [=struct=] with the following items:
 :: A [=boolean=].
 : <dfn>fenced</dfn>
 :: A [=boolean=].
+: <dfn>attribution scope limit</dfn>
+:: Null or a positive 32-bit integer, number of distinct [=attribution source/attribution scopes=] allowed per [=attribution source/attribution destinations|attribution destination=] for the source's [=attribution source/reporting origin=].
+: <dfn>attribution scopes</dfn>
+:: A [=set=] of [=strings=].
+: <dfn>max event states</dfn>
+:: Null or a positive unsigned 32-bit integer, maximum number of [=trigger states=] for [=source type/event=] per [=attribution source/attribution destinations|attribution destination=] for the source's [=attribution source/reporting origin=].
```

Comment:
Force-pushing commits keeps wiping out existing conversions, so I'm reiterating that if this is a uint32, it cannot represent some valid source registrations.

Reply:
Updated this to be 64 bit integer. please lmk if I missed anything in the validator.